packages/connect-node/src/node-universal-client.ts

```diff
@@ -321,12 +321,14 @@ function h2Request(
     session.off("error", sentinel.reject);
     session.off("error", h2SessionConnectError);
     session.on("error", sentinel.reject);
-    sentinel.finally(() => {
-      session.off("error", sentinel.reject);
-      if (!sessionHolder.keepOpen) {
-        session.close();
-      }
-    });
+    sentinel
+      .finally(() => {
+        session.off("error", sentinel.reject);
+        if (!sessionHolder.keepOpen) {
+          session.close();
+        }
+      })
+      .catch(() => {});
```

Comment:
Could you add a comment?

```suggestion
      .catch(() => {
        // We intentionally swallow sentinel rejection - errors must 
        // propagate through the request or response iterables.
      });
```

Reply:
Of course!